test/test_mps.py

```diff
@@ -6940,6 +6940,61 @@ def helper(value, dim, index, idx_dtype=torch.int32):
         with self.assertRaisesRegex(RuntimeError, "Index to scalar can have only 1 value"):
             helper(22, 0, [])
 
+    # TODO: This test can be removed once the backward pass of embedding_bag is
```

Comment:
Hmm, what stops us from running existing forward tests from op_info and just add emebdding_bag to GRAD_FAILURES?